03_12_2020_List/src/telran/OurArrayList.java

```diff
@@ -56,19 +61,119 @@ public int size() {
     }
 
     @Override
-    public void clear() {
+    public void clear() { //O(1)
         source = new Object[INITIAL_CAPACITY];
         size = 0;
     }
 
     @Override
-    public boolean remove() {
+    public boolean remove(T obj) { //O(n^2)
```

Comment:
O(n) - the same complexity as removeById. The number of operations here is linearly dependent on the size (n)